### DISCUSSION TITLE
Welcome to Swift-Radio-Pro Discussions!

### DISCUSSION THREAD

**Discussion by analogcode** [analogcode/Swift-Radio-Pro#174] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bw-development** [comment]
Is that possible to add ability when listening podcasts: "MP3 Chapters Standard",- full support for embedding titles, link URLs, and images, and also so-called "hidden chapters" (special chapters that don’t display in the chapter list (in supported podcast players) — they simply show a link or image to the listener at a relevant time, for whatever duration you’d like) ?

**Reply by bw-development** [reply]
Or this is more to FRadioPlayer creators?
